test-files/indentation-tests.cs

```diff
@@ -159,6 +159,16 @@ public void Test()
             {
                 System.Console.WriteLine("boo");
             }
+
+            /* Callback indentation test. */
+            SomeFunction(() => {
+                    // empty.
+                });
```

Comment:
I don't know how to fix this yet. :/

I was expecting this to be something like.
```cs
SomeFunction(() => {
    // empty.
});
```

Reply:
Yeah I see. Indentation code can be quite tricky. This form is not really idiomatic C# though, so I don't think it's a deal-breaker.